src/application/browser/logger-frontend-module.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (C) 2017 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { ContainerModule } from 'inversify';
+import { WebSocketConnectionProvider } from '../../messaging/browser/connection';
+import { ILogger, Logger } from '../common/logger';
+
+export const loggerFrontendModule = new ContainerModule(bind => {
+    bind(ILogger).toDynamicValue(ctx => {
+        const connection = ctx.container.get(WebSocketConnectionProvider);
+        const rpcProxy = connection.createProxy<ILogger>("/logger");
+        let cachedLevel: number | undefined = undefined;
+
+        const levelFromName: any = {
```

Comment:
Do you need an explicit `any` type here, I prefer to use specific types or let ts compiler infer them